### DISCUSSION TITLE
What version is compatible with Geforce MX110?

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [microsoft/onnxruntime#21273] [General]
Hi,

What version is compatible with Geforce MX110?

I want run this with my GPU:
https://github.com/haofanwang/inswapper

Thanks.

**Comment by carzh** [comment]
I believe Nvidia has information about what versions of CUDA are supported on which hardware. The ONNXRuntime version compatibility tables for different CUDA versions is [here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html)
